recipe/meta.yaml

```diff
@@ -1,25 +1,30 @@
-{% set version = "2.0" %}
+{% set version = "3.1.1" %}
 
 package:
   name: python-gflags
   version: {{ version }}
 
 source:
   fn: python-gflags-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/source/p/python-gflags/python-gflags-{{ version }}.tar.gz
-  md5: 23c9a793959a54971b1f094b0c6d03b1
+  # ugly url, anything better?
+  url: https://pypi.python.org/packages/ea/30/b8469c0d1837ce58fe3706e1f7169cbf6ca1fb87d1f84cece5182b67cb0b/python-gflags-3.1.1.tar.gz
```

Comment:
Please look at the [`example`]( https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml ) recipe and update accordingly.